contract/test/Bank.spec.ts

```diff
@@ -26,88 +26,155 @@ describe('Bank', () => {
     let member1: SignerWithAddress
     let member2: SignerWithAddress
 
-    context('I want to deploy my Bank Smart Contract', () => {
-      before(async () => {
-        ;[owner, contractor, member1, member2] = await ethers.getSigners()
-        await deployContract(owner)
-      })
+    before(async () => {
+      ;[owner, contractor, member1, member2] = await ethers.getSigners()
+      await deployContract(owner)
+    })
 
-      it('Then I become the owner of the contract', async () => {
+    context('Deployment', () => {
+      it('should set the correct owner and tips address', async () => {
         expect(await bankProxy.owner()).to.eq(await owner.getAddress())
+        expect(await bankProxy.tipsAddress()).to.eq(await tipsProxy.getAddress())
       })
+    })
 
-      it('Then the address i provide during the bank contract deployment becomes the address of tips contract', async () => {
-        expect(await bankProxy.tipsAddress()).to.eq(await tipsProxy.getAddress())
+    context('Deposits and Transfers', () => {
+      it('should allow the owner to deposit and transfer funds', async () => {
+        const depositAmount = ethers.parseEther('10')
+        const transferAmount = ethers.parseEther('1')
+
+        await expect(async () =>
+          owner.sendTransaction({ to: await bankProxy.getAddress(), value: depositAmount })
+        ).to.changeEtherBalance(bankProxy, depositAmount)
+
+        const tx = bankProxy.transfer(contractor.address, transferAmount)
+        await expect(tx).to.changeEtherBalance(bankProxy, -transferAmount)
+        await expect(tx).to.changeEtherBalance(contractor, transferAmount)
```

Comment:
Grouping test according to feature, text feature expected result and failing case